setup.py

```diff
@@ -51,8 +48,9 @@
         install_requires=install_requirements,
         extras_require={
             "test": test_requirements,
-            "data": data_requires,
-            "dev": test_requirements + data_requires,
+            "data": data_requirements,
+            "cluster": cluster_requirements,
```

Comment:
As developers we probably want an easy way to install test and cluster without installing data (i.e. be able to build an environment that doesn't rely on the environment limitations of the CC tools), since building artifacts is a smaller part of the dev workflow. 

I have a two proposals:

```python
"test": test_requirements,
"data": data_requirements,
"cluster": cluster_requirements + test_requirements,
"dev": test_requirements + data_requirements + cluster_requirements,
```
or:
```python
"test": test_requirements,
"cluster": cluster_requirements,
"data": data_requirements + cluster_requirements,
"dev": test_requirements + cluster_requirements,
```
If you have another idea feel free to share it.

Reply:
I like the second option better. Though I was just doing `pip install -e .[test, cluster]`